controllers/admin/AdminAccessController.php

```diff
@@ -137,6 +137,7 @@ public function initPageHeaderToolbar()
 
     public function ajaxProcessUpdateAccess()
     {
+        /* @phpstan-ignore-next-line */
```

Comment:
Why? the `_PS_MODE_DEMO_` is defined in defines.inc.php :thinking:

Reply:
Yes. But it's a constant... so it should not be a configuration variable.